cmd/keytransparency-server/frontend.go

```diff
@@ -183,9 +188,17 @@ func main() {
 	if err != nil {
 		glog.Exitf("Failed to load server credentials %v", err)
 	}
-	auth, err := authentication.NewGoogleAuth()
-	if err != nil {
-		glog.Exitf("Failed to create authentication library instance: %v", err)
+
+	var auth authentication.Authenticator
+	if *fakeAuth {
```

Comment:
A switch here would be more appropriate.  I imagine we'll support multiple types of authentication in the future. 

Perhaps the flag could also be modified to be `auth-type=google` by default and `auth-type=fake` for testing.